### PR TITLE
Update u-boot-artik710.bb

### PR DESCRIPTION
**Pull request by koansoftware**
Solve git clone failure

### CODE REVIEW

File: recipes-bsp/u-boot/u-boot-artik710.bb [hunk 1]
```diff
@@ -12,7 +12,7 @@ PROVIDES += "u-boot"
 
 SRCREV_artik710 = "8a9ae34f41a0c992344aa2dd1b1cc40d01161305"
 SRC_URI_artik710 = " \
-    git://git@github.com/resin-os/uboot-artik7.git;protocol=ssh \
+    git://github.com/resin-os/uboot-artik7.git \
```
Comment (floion) [inline recipes-bsp/u-boot/u-boot-artik710.bb]: Hi. Can you paste the error you get without this change?

**Comment by floion** [comment]
Have you seen this error when trying to clone this repository separately or when you were trying to build the resin-artik710?

**Comment by koansoftware** [comment]
Both

**Comment by koansoftware** [comment]
Would you mind to apply them to 'morty' branch as well ?

**Comment by agherzan** [comment]
How did this work in the past? As well it would should use yocto guidelines for git commit format.

**Pull request merged**